devtools/dump_devinfo.py

```diff
@@ -207,6 +233,52 @@ async def handle_device(basedir, autosave, device: Device, batch_size: int):
         + " Do not use this flag unless you are sure you know what it means."
     ),
 )
+@click.option(
+    "--discovery-timeout",
+    envvar="KASA_DISCOVERY_TIMEOUT",
+    default=10,
+    required=False,
+    show_default=True,
+    help="Timeout for discovery.",
+)
+@click.option(
+    "-e",
+    "--encrypt-type",
+    envvar="KASA_ENCRYPT_TYPE",
+    default=None,
+    type=click.Choice(ENCRYPT_TYPES, case_sensitive=False),
+)
+@click.option(
+    "-df",
+    "--device-family",
+    envvar="KASA_DEVICE_FAMILY",
+    default="SMART.TAPOPLUG",
+    help="Device family type, e.g. `SMART.KASASWITCH`. Deprecated use `--type smart`",
+)
+@click.option(
+    "-lv",
+    "--login-version",
+    envvar="KASA_LOGIN_VERSION",
+    default=2,
+    type=int,
+    help="The login version for device authentication. Defaults to 2",
+)
+@click.option(
+    "--https/--no-https",
+    envvar="KASA_HTTPS",
+    default=False,
+    is_flag=True,
+    type=bool,
+    help="Set flag if the device encryption uses https.",
+)
+@click.option(
+    "--experimental",
+    default=False,
+    is_flag=True,
+    type=bool,
+    envvar="KASA_EXPERIMENTAL",
+    help="Enable experimental mode for devices not yet fully supported.",
```

Comment:
Given this is a devtool, maybe we could enable experimental always to keep the code cleaner?

Reply:
Done